includes/class-wc-payments-blocks-payment-method.php

```diff
@@ -71,13 +71,7 @@ public function get_payment_method_script_handles() {
 			true
 		);
 
-		$script_dependencies = [ 'stripe' ];
-
-		if ( WC_Payments_Features::is_tokenized_cart_ece_enabled() && ( is_cart() || is_checkout() || is_product() || has_block( 'woocommerce/checkout' ) || has_block( 'woocommerce/cart' ) ) ) {
-			$script_dependencies[] = 'WCPAY_PAYMENT_REQUEST';
-		}
-
-		WC_Payments::register_script_with_dependencies( 'WCPAY_BLOCKS_CHECKOUT', 'dist/blocks-checkout', $script_dependencies );
```

Comment:
This is how the PRBs were enqueued. The ECE is enqueued differently, we're keeping it that way.
So this block with the `WCPAY_PAYMENT_REQUEST` script has become deprecated, and can now be simplified.